superset/sql_parse.py

```diff
@@ -144,10 +144,11 @@ def __extract_from_token(self, token, depth=0):
 
             if table_name_preceding_token:
                 if isinstance(item, Identifier):
-                    self.__process_identifier(item)
+                    self.__process_tokenlist(item)
                 elif isinstance(item, IdentifierList):
                     for token in item.get_identifiers():
-                        self.__process_identifier(token)
+                        if isinstance(token, TokenList):
```

Comment:
There's no guarantee that the "identifiers" associated withe [`get_indentifiers`](https://github.com/andialbrecht/sqlparse/blob/master/sqlparse/sql.py#L470) (possibly poorly named) are of type `Identifier` and thus the additional check to verify that the token is an instance of `TokenList` is required.